modules.json

```diff
@@ -12,7 +12,6 @@
   "ozoneBidAdapter",
   "permutiveRtdProvider",
   "prebidServerBidAdapter",
-  "pubCommonId",
```

Comment:
Module was deprecated and removed here https://github.com/prebid/Prebid.js/pull/9869 I'm not sure if we use it?

Reply:
@Amouzle confirmed this module is no longer required